src/app/shared/services/tags/tag.service.ts

```diff
@@ -54,13 +54,13 @@ export class TagService extends BaseBackendCachedService<Tag> {
     })
       .map(() => {
         if (newEntity.tags) {
-         const newTags = Object.assign([], newEntity.tags);
+          const newTags = { ...newEntity.tags };
```

Comment:
newTags should be array or tags.
[...newEntity.tags]

and why do you need to copy that array?